src/main/java/de/oliver/fancyholograms/HologramManagerImpl.java

```diff
@@ -160,7 +174,7 @@ public void loadHolograms(String world) {
      */
     void initializeTasks() {
         ScheduledExecutorService hologramThread = plugin.getHologramThread();
-        hologramThread.schedule(() -> {
+        Bukkit.getScheduler().runTask(plugin, () -> {
```

Comment:
I think this should be kept on the hologram thread

Reply:
Since we're now firing an event for loaded holograms I think that's alright

Reply:
This would no longer be safe as this is the main thread, I believe the method would be `hologramThread#submit`